### DISCUSSION TITLE
PCIe Coral in hass.io add-on issue

### DISCUSSION THREAD

**Discussion by pflumph** [blakeblackshear/frigate#934] [Ask A Question] [answered]
I've installed the PCIe Coral in my Debian Home Assistant server. It's tested VIA CLI and working.
Frigate add-on is installed in HA as well, but logs show fail:
frigate.edgetpu                INFO    : Attempting to load TPU as usb
Process detector:coral:
frigate.edgetpu                INFO    : No EdgeTPU detected.

Presumably because the Coral is at /dev/apex_0 instead of /dev/bus/usb
This isn't configurable in the GUI configuration tab, so I would expect there's a config I need to modify.
Where might that be?

Thank you

**Comment by pflumph** [comment, marked as answer]
Fixed:
Add the following to frigate.yaml
detectors:
coral:
type: edgetpu
device: pci

**Reply by HIT2022** [reply]
In my PCI case at least, I have to disable protection on the hassio add-on, in order for the add-on to detect coral.

**Reply by pflumph** [reply]
I was more shocked that it ran at all as my Lenovo Tiny is notorious for only allowing whitelisted PCIE devices
